geometry_msgs/Point futureTrajectory
---
float64 intDist